driver-core/src/main/com/mongodb/client/model/expressions/package-info.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * API for MQL expressions.
+ *
+ * @see com.mongodb.client.model.expressions.Expression
+ */
+@NonNullApi
+package com.mongodb.client.model.expressions;
```

Comment:
We need to document the server version requirement for each methods in the new API. This requirement depends on the MQL used by the implementation of a method, and the implementation is allowed be changed in the future only in such ways that either don't increase the server version requirement, or increase it up to the least supported server version.

Reply:
Should be done as of https://github.com/mongodb/mongo-java-driver/pull/1070